tests/Microsoft.Identity.Test.Unit/AppConfigTests/PublicClientApplicationBuilderTests.cs

```diff
@@ -5,9 +5,7 @@
 using System.IO;
 using System.Linq;
 using Microsoft.Identity.Client;
-#if !NET6_WIN && !NET7_0 && !NET6_0_OR_GREATER
```

Comment:
Should this be
#if  !NET7_0 && !NET6_0_OR_GREATER
using Microsoft.Identity.Client.Desktop;
?

does !NET6_0_OR_GREATER include Net6 win?

Reply:
It does include it, but to your point NET6_WIN is gone completely. THe unit test is built against NET7